eth/tools/_utils/slow_code_stream.py

```diff
@@ -65,27 +65,27 @@ def __next__(self) -> int:
             return STOP
 
     def peek(self) -> int:
-        current_pc = self.pc
+        current_pc = self.program_counter
```

Comment:
I have this in a separate commit but thought I could sneak it in here. I personally had to lookup what `pc` stands for again and I remember I already looked it up in the past. Part of this is probably just my poor memory but I also think it aligns better with the rest of the codebase to not use two letter acronyms :)